### ISSUE TITLE
Support nsis3

### ISSUE THREAD

**Issue by kenhys** [clear-code/fx-meta-installer#15]
## Problem

In debian/unstable, default version of nsis is changed to nsis3.
As a result, it doesn't work.

# Expected

make succeeds without error.

# Actual 

```
% mkdir nsis3
% git clone git@github.com:clear-code/fx-meta-installer.git
% ./fx-meta-installer/scripts/scafold.sh nsis3
% ./fx-meta-installer/scripts/make_installer.sh nsis3
```

Then, Bad text encoding occurs.

```
./fx-meta-installer/scripts/make_installer.sh nsis3
Executing: rm -rf /home/kenhys/work/firefox/nsis3/fx-meta-installer/resources [/home/kenhys/work/firefox/nsis3]
 => Succeeded

Executing: mkdir /home/kenhys/work/firefox/nsis3/fx-meta-installer/resources [/home/kenhys/work/firefox/nsis3]
 => Succeeded

Executing: rm -rf /home/kenhys/work/firefox/nsis3/fx-meta-installer/..//dist//nsis3-source [/home/kenhys/work/firefox/nsis3]
 => Succeeded

Executing: mkdir /home/kenhys/work/firefox/nsis3/fx-meta-installer/..//dist/ [/home/kenhys/work/firefox/nsis3]
mkdir: cannot create directory '/home/kenhys/work/firefox/nsis3/fx-meta-installer/..//dist/': File exists
 => Failed to create directory. Proceed anyway.

Executing: /home/kenhys/work/firefox/nsis3/fx-meta-installer/scripts/make_addons.sh [/home/kenhys/work/firefox/nsis3]
Executing: rm -r /home/kenhys/work/firefox/nsis3/fx-meta-installer/resources [/home/kenhys/work/firefox/nsis3]
 => Succeeded

Executing: mkdir /home/kenhys/work/firefox/nsis3/fx-meta-installer/resources [/home/kenhys/work/firefox/nsis3]
 => Succeeded

 => Succeeded

Executing: cp -t /home/kenhys/work/firefox/nsis3/fx-meta-installer/ config.bat config.nsh fainstall.ini [/home/kenhys/work/firefox/nsis3/installer-config]
 => Succeeded

Executing: ./make.sh [/home/kenhys/work/firefox/nsis3/fx-meta-installer]
nsis3
rm: cannot remove 'fainstall.exe': No such file or directory
rm: cannot remove 'nsis3.exe': No such file or directory
rm: cannot remove 'nsis3-*.exe': No such file or directory
Processing config: /etc/nsisconf.nsh
Processing script file: "fainstall.nsi" (UTF8)
Bad text encoding: ../config.nsh:12
!include: error in script: "../config.nsh" on line 12
Error in script "fainstall.nsi" on line 86 -- aborting creation process
Failed to build fainstall.exe!
 => Failed
nsis3 rm: cannot remove 'fainstall.exe': No such file or directory rm: cannot remove 'nsis3.exe': No such file or directory rm: cannot remove 'nsis3-*.exe': No such file or directory Processing config: /etc/nsisconf.nsh Processing script file: "fainstall.nsi" (UTF8) Bad text encoding: ../config.nsh:12 !include: error in script: "../config.nsh" on line 12 Error in script "fainstall.nsi" on line 86 -- aborting creation process Failed to build fainstall.exe!
Executing: cp /home/kenhys/work/firefox/nsis3/fx-meta-installer/../installer-config/fainstall.ini /home/kenhys/work/firefox/nsis3/fx-meta-installer/nsis3-source/ [/home/kenhys/work/firefox/nsis3]
cp: cannot create regular file '/home/kenhys/work/firefox/nsis3/fx-meta-installer/nsis3-source/': Not a directory
 => Failed
cp: cannot create regular file '/home/kenhys/work/firefox/nsis3/fx-meta-installer/nsis3-source/': Not a directory
```

**Comment by kenhys** [comment]
I've created a branch for nsis3.

https://github.com/kenhys/fx-meta-installer/tree/support-nsis3

Currently, makensis succeeds, but built fxmetainstaller.exe can't work as expected.
It seems that FxMetaInstaller.exe can't get admin priviledge correctly.

**Comment by piroor** [comment]
I've opened #23 which should solve this.

**Comment by kenhys** [comment]
> https://github.com/kenhys/fx-meta-installer/tree/support-nsis3

Above branch is deprecated because there is needless commit (since 3.07 unicode configuration is changed to `true` by default)

**Comment by piroor** [comment]
I close this because fixed by #23.
